scalameta/metals
# Fix Mercury blog post date by adding an unlisted redirect

This uses a new feature of Docusaurus 1.12.0, contributed by @olafurpg 🎉, which allows marking blog posts as unlisted.

With this, we can:

- republish the faulty blog post with the correct date
- make the old post point to the new one and make it unlisted

This way old links will continue to work.

**NOTE**
During development, unlisted blog posts are still visibile, so we will still see the "redirect" blog post when doing `yarn start`.

To test this change, I did:

```
yarn build
npx http-server build
```

![image](https://user-images.githubusercontent.com/691940/61621313-01a02600-ac73-11e9-81f4-740a48089c15.png)

This is what the old link will point to. Notice the absence of this particular entry in the sidebar.
